app/assets/stylesheets/revised/pages/landing/_home.scss

```diff
@@ -281,11 +432,13 @@
 }
 
 .root-landing-who {
+  background-color: #f9f9f9;
```

Comment:
I think, because of the white background in the images, it would be great if the background here was white too.

The quick and easy fix I see that makes the alternating background still exist is switching the "how to" to be `#f9f9f9` and then making this and the recoveries white. It looks like you'd have to add some padding to the bottom of the "how to" section.

Of course, if you see something easier go for that instead. I think the alternating background is nice and subtly breaks up the page and so makes sense to continue.

Reply:
Updated in 6aa173b5